modules/admin-ui-picard/app/src/thunks/userDetailsThunks.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import {loadUserDetailsFailure, loadUserDetailsInProgress, loadUserDetailsSuccess} from "../actions/userDetailsActions";
+import {addNotification} from "./notificationThunks";
+import {buildUserBody} from "../utils/resourceUtils";
+
+
+// fetch details about certain user from server
+export const fetchUserDetails = username => async dispatch => {
+    try {
+        dispatch(loadUserDetailsInProgress());
+
+        let data = await axios.get(`admin-ng/users/${username}.json`);
+
+        const response = await (data.data);
+
+        const userDetails = {
+            ...response,
+            email: !!response.email ? response.email : ''
+        }
+
```

Comment:
add Notification (on top of the tab), if user is not manageable (see comment on EditUserGeneralTab)

Reply:
`manageable` is part of `response` and after change applied to `EditUserGeneralTab` (see above) this change is no longer needed.